tests/test_deep_reference_parser.py

```diff
@@ -17,10 +18,45 @@ def tmpdir(tmpdir_factory):
     return tmpdir_factory.mktemp("data")
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="session")
 def cfg():
-    return get_config(TEST_CFG)
+    cfg = get_config(TEST_CFG)
+
+    artefacts = [
+        "char2ind.pickle",
+        "ind2label.pickle",
+        "ind2word.pickle",
+        "label2ind.pickle",
+        "maxes.pickle",
+        "weights.h5",
+        "word2ind.pickle",
+    ]
+
+    S3_SLUG = cfg["data"]["s3_slug"]
+    OUTPUT_PATH = cfg["build"]["output_path"]
+    WORD_EMBEDDINGS = cfg["build"]["word_embeddings"]
+
+    for artefact in artefacts:
+        with msg.loading(f"Could not find {artefact} locally, downloading..."):
+            try:
+                artefact = os.path.join(OUTPUT_PATH, artefact)
+                download_model_artefact(artefact, S3_SLUG)
```

Comment:
Could this be marked as an integration test or something to make explicit that it depends on downloading stuff from S3?

Reply:
yep good idea. I'll ad a pytest mark

Reply:
Resolved in https://github.com/wellcometrust/deep_reference_parser/pull/21/commits/e7d93d70d6490ead36276dadcbaa235841472d49